test/stream-to-buffer.js

```diff
@@ -0,0 +1,14 @@
+const streamToBuffer = async (readableStream) => {
```

Comment:
This is redundant if you use `blobClient.downloadToBuffer()` instead of `blobClient.download()`.

Reply:
Removed